bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/entity/ChangelogEntity.kt

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright (c) 2021,2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package org.eclipse.tractusx.bpdm.gate.entity
+
+import jakarta.persistence.*
+import org.eclipse.tractusx.bpdm.common.model.BaseEntity
+import org.eclipse.tractusx.bpdm.gate.dto.response.LsaType
+
+
+@Entity
+@Table(name = "changelog_entries")
+class ChangelogEntity (
+
+    @Column(name = "external_id", nullable = false, updatable = false)
+    val externalId: String,
+    @Enumerated(EnumType.STRING)
+    @Column(name = "business_partner_type", nullable = false, updatable = false)
+    val businessPartnerType: LsaType
```

Comment:
Same here with the LsaType. Better create a new enumeration which only contains the actual three LSA types. 
Suggestion: rename this LsaType to OptionalLsaType and introduce a new LsaType enumeration with only those three types as options.